pkg/api/types.go

```diff
@@ -195,6 +195,119 @@ type VolumeSource struct {
 	NFS *NFSVolumeSource `json:"nfs"`
 }
 
+// Similar to VolumeSource but meant for the administrator who creates PVs.
+// Exactly one of its members must be set.
+type PersistentVolumeSource struct {
```

Comment:
@thockin @bgrant0607 suggested this distinction.  Not all volumes are persistent, only a subset.  Volumes may  overlap between the two.  Kubelet has to be made to work with both, which is what Tim was prepping for in #5642.

I tend to agree with you because the implementation is simpler, but then someone just has to ask why an EmptyDir can be a persistent volume (just as I'm using HostPath as a persistent volume for development).